src/Entity/Media/Image.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of AppName.
+ *
+ * (c) Monofony
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Entity\Media;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\MappedSuperclass
+ */
+abstract class Image extends File
```

Comment:
Are you sure this additional abstraction layer is needed?

Reply:
You're right, I'll just keep the more generic "File" entity.